parameters.hh

```diff
@@ -69,9 +71,18 @@ namespace global_parameters {
     bool gradient = false; // wether to measure the gredient flow or not
     double tmax = 1.0; // tmax for gradient flow"
     std::string confdir = "./"; // directory where gauge configurations are stored
+    
+    bool potential = false; //measure potential: the loops W(x, t, y=z=0) and W(x, y, t=z=0) are measured with a maximum size of lattice extent * sizeloops, and written to separate files. Only available for ndims=3,4
+    bool potentialsmall = false; //The loops W(x, t, y) are measured up to x, y=min(4, lattice extent), t <= Lt * sizeloops and saved to one file. Only available for ndim=3
+    bool append = false; //are measurements for potential appended to an existing file, or should it be overwritten?
+    double sizeloops = 0.5; //Wilson-Loops are measured up to this fraction of the lattice extent
+    size_t n_apesmear = 0; //number of APE smearings done on the lattice before measurement. 
+    //APE-smearing is done before measuring the potential and small potential, it does not affect the gradient flow and Wilson-loops
+    double alpha = 1.0; //parameter alpha for APE smearings
+    bool smearspacial = false; //should smearing be done only for spacial links?
```

Comment:
I suggest to change the name to ```smear_spatial_only```

Reply:
I agree that would make it more clear, then I would also change spacial to spatial_only